.travis.yml

```diff
@@ -16,12 +22,17 @@ matrix:
       otp_release: 19.3
     - elixir: 1.8
       otp_release: 20.3
+
 sudo: false
+
 env:
   global:
     - ELIXIR_ASSERT_TIMEOUT=2000
+
 before_script:
   - mix deps.get
   - nvm install 6.2 && nvm use 6.2
```

Comment:
This should be `nvm install 8.9 && nvm use 8.9`. You can use a higher/more current minor version, but 8.9 was the first LTS release for v8 IIRC.